test/components/position_component_test.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:math' as math;
+
+import 'package:flame/anchor.dart';
+import 'package:flame/components/position_component.dart';
+import 'package:flame/extensions/vector2.dart';
+import 'package:test/test.dart';
+
+class MyComponent extends PositionComponent {}
+
+void main() {
+  group('PositionComponent overlap test', () {
+    test('overlap', () {
+      final PositionComponent component = MyComponent();
+      component.position = Vector2(2.0, 2.0);
+      component.size = Vector2(4.0, 4.0);
```

Comment:
and below: `all`

Reply:
All of the tests in here follow the same standard even if they happen to have the same arguments so that it is easy to modify them.